x-pack/platform/plugins/shared/onechat/common/http_api/chat.ts

```diff
@@ -20,6 +21,7 @@ export interface ChatRequestBodyPayload {
   conversation_id?: string;
   capabilities?: AgentCapabilities;
   input: string;
+  browserApiTools?: BrowserApiToolMetadata[];
```

Comment:
1. Should use snake case (for all our APIs and domain model - I know it's hard 😄 )
2. We need a better name (`client_actions` ? I can even accept `client_tools` if we really want)